velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -99,6 +106,51 @@ void readData(ReadFile* readFile) {
   ASSERT_EQ(warfFromBuf, "abbbbbcc");
 }
 
+HdfsWriteFile openHdfsWriteFile(std::string_view path) {
+  struct hdfsBuilder* builder = hdfsNewBuilder();
+  hdfsBuilderSetNameNode(builder, localhost.c_str());
+  hdfsBuilderSetNameNodePort(builder, atoi(hdfsPort.c_str()));
+  auto hdfsClient = hdfsBuilderConnect(builder);
```

Comment:
These lines repeat multiple times in this file. Consider extracting a helper function to reduce copy-paste.